explorer/src/components/instruction/InstructionCard.tsx

```diff
@@ -23,10 +25,20 @@ export function InstructionCard({
   index,
   ix,
   defaultRaw,
+  fetchRawTrigger,
```

Comment:
Rather than passing a fetch trigger down, perhaps we could just set up useTransactionDetails and useFetchRawTransaction inside the InstructionCard itself. This would require passing down the signature either through props or establishing it in context (up to you, but props would be sufficient).